package.json

```diff
@@ -17,6 +17,17 @@
             "url": "https://github.com/notanengineercom"
         }
     ],
+    "repository": {
+        "type" : "git",
+        "url" : "https://github.com/ffMathy/FluffySpoon.JavaScript.Testing.Faking.git"
+    },
```

Comment:
This was what originally drove me to create this PR. I was upgrading this package for a client and tried to run `npm repo` for this package:

```console
> npm repo @fluffy-spoon/substitute
npm ERR! no repository
```

By adding this block, users can now get directly to your GitHub project by running `npm repo @fluffy-spoon/substitute`.